daemon/create.go

```diff
@@ -14,26 +14,35 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// ContainerCreate is the parameter set to ContainerCreate()
+type ContainerCreate struct {
+	Name            string
+	Config          *runconfig.Config
+	HostConfig      *runconfig.HostConfig
+	AdjustCPUShares bool
+	FromBuilder     bool
```

Comment:
It still has to create the volume, and all volumes do get names, just a matter of if the names is generated or the name is specified by the user. In cases where the user specifies the name, it will only be done through binds.